lib/SPIRV/libSPIRV/SPIRVDecorate.cpp

```diff
@@ -114,6 +114,9 @@ void SPIRVDecorate::encode(spv_ostream &O) const {
   case DecorationUserSemantic:
     SPIRVDecorateUserSemanticAttr::encodeLiterals(Encoder, Literals);
     break;
+  case spv::internal::DecorationHostAccessINTEL:
```

Comment:
Seems like there should be
`switch (static_cast<uint32_t>(Dec)) {`

Reply:
Thanks! Found that the same fix was applied for `SPIRVDecorate::getRequiredExtension()`, will apply that.